met-web/src/store.ts

```diff
@@ -4,13 +4,15 @@ import userSlice from 'services/userService/userSlice';
 import { form, submission } from '@formio/react';
 import engagementSlice from 'services/engagementService/engagementSlice';
 import notificationSlice from 'services/notificationService/notificationSlice';
+import modalSlice from 'services/modalService/modalSlice';
 
 export const store = configureStore({
     reducer: {
         auth: loginReducer,
         user: userSlice,
         engagement: engagementSlice,
         notification: notificationSlice,
+        modal: modalSlice,
```

Comment:
more specific name, notificationModalSlice or better

Reply:
we might use other custom modals, so let's make sure that this is classified as something specific for notifications or something